files/es/web/css/css_container_queries/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: CSS Container Queries
+slug: Web/CSS/CSS_Container_Queries
+page-type: guide
+tags:
+  - CSS
+  - CSS Containment
+  - CSS Container Queries
+  - Guide
+  - Paint
+  - Responsive Design
+l10n:
+  sourceCommit: 674fd4fb4225dfad82eab7425f9fc8e52d67a7db
+---
+
+{{CSSRef}}
+
+[CSS containment](/es/docs/Web/CSS/CSS_Containment) proporciona una nueva forma para aislar secciones de una página e indicarle al navegador que estas secciones son independientes del resto de la página en términos de estilos y disposición (_layout_).
+
+Si estás creando un [diseño responsivo](/es/docs/Learn/CSS/CSS_layout/Responsive_Design), a menudo usas [media queries](/en-US/docs/Web/CSS/Media_Queries) para cambiar la disposición de la página con referencia al {{Glossary("viewport")}}.
```

Comment:
- Due to it is an English concept
- All links needs to point to `/es/` exists it or not, the bot automatically redirect

```suggestion
Si estás creando un [diseño responsivo](/es/docs/Learn/CSS/CSS_layout/Responsive_Design), a menudo usas [_media queries_](/es/docs/Web/CSS/Media_Queries) para cambiar la disposición de la página con referencia al {{Glossary("viewport")}}.
```